Sources/TuistLoader/Loaders/TemplateLoader.swift

```diff
@@ -27,17 +27,18 @@ public class TemplateLoader: TemplateLoading {
 
     public func loadTemplate(at path: AbsolutePath) throws -> TuistCore.Template {
         let template = try manifestLoader.loadTemplate(at: path)
+        let generatorPaths = GeneratorPaths(manifestDirectory: path)
         return try TuistCore.Template.from(manifest: template,
-                                           at: path)
+                                           generatorPaths: generatorPaths)
```

Comment:
Nice! So we can use now relative paths 🎉

Reply:
Yes! This should make it easier for users to define templates with shared template files.